ds_judgements_public_ui/templates/includes/search_term_component.html

```diff
@@ -1,7 +1,15 @@
-{% load static search_results_filters %}
-<div class="search-term-component__search-term-container">
-  <label for="search_term" class="search-term-component__search-term-label">Search</label>
-  {{ form.query }}
-  {% include "includes/search/search_help_text.html" %}
-</div>
-<input type="submit" value="Search" />
+{% load static search_results_filters waffle_tags %}
+{% flag 2_homepage %}
+  <div class="search-term-component__search-term-container">
+    <label for="search_term" class="search-term-component__search-term-label">Search</label>
+    {{ form.query }}
+    {% include "includes/search/search_help_text.html" %}
```

Comment:
I think we can remove this on the variation - it's not included in the design

Reply:
@jlhdxw thanks this was supposed to not be behind the flag as its on the current homepage as help text so have taken it out of the flag variation and put back